internal/analytics/analytics.go

```diff
@@ -80,10 +81,18 @@ func (d *customDimensions) toMap() map[string]string {
 	}
 }
 
+var (
+	eventWaitGroup sync.WaitGroup
+)
+
 func init() {
 	setup()
 }
 
+func WaitForAllEvents() {
+	eventWaitGroup.Wait()
```

Comment:
Could we add a timeout to this? Let's wait for at most a second. I don't want slow connections or firewalls to cause commands to just hang. And loosing out on events from those outliers should be pretty negligible.

Reply:
To add the timeout, I had to change our google-analytics library for this PR: 

So, we would need to merge this one first: https://github.com/ActiveState/go-ogle-analytics/pull/1, and then update the go.mod dependencies here.